Libplanet/Tx/TxId.cs

```diff
@@ -20,9 +20,10 @@ namespace Libplanet.Tx
     /// (See also <see cref="Size"/> constant.)</para>
     /// </summary>
     /// <seealso cref="Transaction{T}.Id"/>
-    [JsonConverter(typeof(TxIdJsonConverter))]
```

Comment:
Is this removed because Bencodex can handle Json marshalling?

Reply:
This isn't removed. It's just `git diff` acting up. 🙃